conf.d/docker-compose.yml

```diff
@@ -19,6 +19,7 @@ services:
     container_name: FLUENTD
     ports:
       - "20000-20010:20000-20010/tcp"
+      - 20001:20001/udp
```

Comment:
20001  is included in 20000-20010, how about  deleting  `/tcp` directly?

Reply:
Docker port mapping defaults to tcp, so /tcp can be deleted, but 20001 must be specified as udp.

Reply:
> Docker port mapping defaults to tcp, so `/tcp` can be deleted, but 20001 must be specified as `udp`.

Does it mean if we use `- 20000-20010:20000-20010` only, `udp` on 20001 will not work?

Reply:
> > Docker port mapping defaults to tcp, so `/tcp` can be deleted, but 20001 must be specified as `udp`.
> 
> Does it mean if we use `- 20000-20010:20000-20010` only, `udp` on 20001 will not work?

Yes.

Reply:
Got it. Thanks.